packages/terra-menu/tests/jest/__snapshots__/Menu.test.jsx.snap

```diff
@@ -524,6 +524,7 @@ exports[`Menu correctly applies the theme context className 1`] = `
                                                   "timeZone": null,
                                                 }
                                               }
+                                              isTabFocusDisabled={false}
```

Comment:
Thank you for taking care of that! Came across the same problem in other PR, glad to see it's taken care of!